src/python_framework/nhd_network_traversal.py

```diff
@@ -2,34 +2,60 @@
 A demonstration version of this code is stored in this Colaboratory notebook:
     https://colab.research.google.com/drive/1ocgg1JiOGBUl3jfSUPCEVnW5WNaqLKCD
 
+
+## command line input order [debuglevel = 0,verbose = true,showtiming = true, supernetworks = supernetwork_options]
 """
 import nhd_network_utilities as nnu
 import recursive_print
 import time
 import os
+import sys
 
-# NOTE: these methods can lose the "connections" and "rows" arguments when
-# implemented as class methods where those arrays are members of the class.
-
+try:
+    arg1 = int(sys.argv[1])
+    arg2 = sys.argv[2]
+    arg3 = sys.argv[3]
+    arg4 = sys.argv[4]
+except:
+    arg1 = 0
+    arg2 = True
+    arg3 = True
+    arg4 = 'Brazos_LowerColorado_ge5'
```

Comment:
Could we just pass these directly into variables with the given names (`debuglevel`,`verbose`, `showtiming`, `supernetwork`... rather than arg1,2,...)?